packages/v4/src/content/design-guidelines/content/writing-design-documentation.md

```diff
@@ -0,0 +1,132 @@
+
+Example:
+
+![Two call outs highlight multiple items on a toolbar.](./img/desguidelines2.png)
+
+**1. Items in view:** Allows the user to select the item count (number of listed items) per page, as seen in full pagination
+
+**2. Compact pager:** Supplies the user with page-back and page-next controls only
+
+Avoid using a callout to highlight a single item, unless you're pointing our a small part of a large view, or the item isn't clearly shown in your image. Use a blank, unnumbered callout to mark the area, then bold it in your explanation.
+
+Example:
+
+![#A single call out highlights the login button on a large screen.](./img/desguidelines3.png)
+
+Always write your **login button** as two words: "Log in."
+
+## Style, voice, and tone
+
+PatternFly's documentation is friendly and instructional. Learn more about voice in our [brand voice and tone section](/ux-writing/brand-voice-and-tone/).
+
+When writing design guidelines, use:
+
+- Present tense.
+- Second-person point of view ("you").
+- Active voice, unless you're using passive voice for emphasis.
+- Direct language and sentence structure.
+- Full words to introduce examples, instead of their abbreviations ("for example" instead of "e.g."), as seen on our [terms list](/ux-writing/terminology/).
+- Descriptive hyperlinks (not "click here").
+Descriptive language instead of directional language (up, down, left, right) to refer to elements in your document.
+
+## Design guidelines template
+
+Use this template asa guide to structure your document. Depending on your content, you might have different sections and headings.
+
+### Design guideline name
+
+Title your page with your component or pattern name. Begin with a short 1-2 sentence description or definition of the design problem, topic, or function addressed in your document.
+
+Example:
+
+"**Pagination** splits data across multiple pages, typically within a table, list, or card view. If you use infinite scrolling, you don't need to use pagination."
+
+### Elements
+
+Introduce your component and the primary elements that compose it. List them and include explanatory copy for each one.
+
+### Usage
+
+Inform your readers about how and when to use this component or pattern. Include an explanation of the problems it solves along with the variations and best practices PatternFly users should be aware of when applying it in their designs.
+
+This section might have subheadings that give further details about using your component in context.
+
+### Content
+
+Provide guidance for writing content that might appear in your component or pattern. This content section, if applicable, demonstrates how general terminology and wording best practices carry over to component-specific guidelines.
+
+Content guidance includes:
+
+- Editorial guidelines for labeling and message text.
+- Length restrictions and what to do when text overflows.
+- Localization considerations.
+
+### Related components and demos
```

Comment:
I don't think we're doing this anymore! @mcarrano can you confirm?

Reply:
Right. We decided that this was no longer needed since now the component documentation, design guidelines and demo pages are all grouped together.

Reply:
Great, I'll remove this section. Thank you!